test/extended/quota/clusterquota.go

```diff
@@ -27,7 +29,23 @@ var _ = g.Describe("[sig-api-machinery][Feature:ClusterResourceQuota]", func() {
 
 	g.Describe("Cluster resource quota", func() {
 		g.It(fmt.Sprintf("should control resource limits across namespaces"), func() {
-			t := g.GinkgoT()
+			t := g.GinkgoT(1)
+
+			versionInfo, err := oc.KubeClient().Discovery().ServerVersion()
+			if err != nil {
+				t.Fatalf("unexpected error: %v", err)
+			}
+			const kubeRootCAName = "kube-root-ca.crt"
+			version, err := utilversion.ParseSemantic(versionInfo.String())
+			if err != nil {
+				t.Fatalf("unexpected error: %v", err)
+			}
+			expectKubeRootCACM := version.AtLeast(utilversion.MustParseGeneric("1.20"))
```

Comment:
what does this mean for downgrades or upgrades?

Reply:
I would rather unconditionally ignore the kube-root-ca configmap.

Reply:
it a server version so if you run it before or after upgrade it won't matter. we don't run tests during upgrades afaik.